container-scan/action.yaml

```diff
@@ -83,12 +83,12 @@ runs:
         github-token: ${{ inputs.github-token }}
         image-platform: ${{ inputs.image-platform }}
         image-version: ${{ inputs.image-tag }}
-        build-args: ${{ inputs.build-args }}
-        secrets: |
+        build-args: |
           ARTIFACTORY_USERNAME=${{ inputs.artifactory-username }}
```

Comment:
Not for now, but probably these could be moved out of this action into having the repos that need it pass it down